.gitignore

```diff
@@ -1,8 +1,9 @@
-target
-.project
-.classpath
-.settings
-.idea
-*.iml
-*.chain
-*.spvchain
+target
+.project
+.classpath
+.settings
+.idea
+*.iml
+*.chain
+*.spvchain
+test.wallet

```

Comment:
Please don't add any .wallet files to .gitignore, even if it's called test.wallet.